main.go

```diff
@@ -1,6 +1,7 @@
 package main
 
 import (
+	validation2 "github.com/stackrox/istio-cves/validation"
```

Comment:
move this down with the other `github.com` imports and also remove the name `validation2`